.github/workflows/build.yml

```diff
@@ -65,34 +65,82 @@ jobs:
       run: echo 'cmake --build ${{github.workspace}}/build --target bootcd --target livecd' | ${{github.workspace}}/RosBE-CI/RosBE.sh . 0 ${{matrix.arch}}
     - name: Print ccache statistics
       run: ccache -s
-    - name: Upload bootcd
+    - name: Upload ISOs
```

Comment:
For what it's worth, I prefer separate ISOs: smaller to download and easier to process, as I often need one of the CDs only.
For the same reason, I will probably not like to have the msvc pdb included in the same archive...